### PR TITLE
update disconnected and proxy for 4.15 dr test

### PR DESCRIPTION
**Pull request by geliu2016**

### CODE REVIEW

**Comment by geliu2016** [comment]
@jianlinliu could you please take quick review? thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml [hunk 1]
```diff
@@ -2436,7 +2436,7 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
-- as: gcp-ipi-private-f28-disasterrecovery
+- as: gcp-ipi-private-proxy-f28-disasterrecovery
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2439]: What is your test strategy? change the existing ones to cover the new configurations, or add new configuration coverage?

Comment (geliu2016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2439]: change existing one to cover more configuration matric

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2439]: get it.

Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2439]:
```suggestion
- as: gcp-ipi-proxy-private-f28-disasterrecovery
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml [hunk 2]
```diff
@@ -2446,8 +2446,8 @@ tests:
     test:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
-    workflow: cucushift-installer-rehearse-gcp-ipi-private
-- as: gcp-ipi-private-fips-f28-disasterrecovery
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-private
+- as: gcp-ipi-private-fips-disconnected-f28-disasterrecovery
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml:2450]:
```suggestion
- as: gcp-ipi-disconnected-private-fips-f28-disasterrecovery
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml (hunk 1 above)
Comment (geliu2016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml]: updated.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml (hunk 2 above)
Comment (geliu2016) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml]: updated.

**Comment by geliu2016** [comment]
@liangxia could you pls take a view?

**Pull request merged**